frontend/error.vue

```diff
@@ -0,0 +1,42 @@
+<template>
+  <div class="bg-[url(public/background.png)]" h-screen w-screen bg-cover bg-no-repeat>
+    <div h-full w-full flex flex-col items-center justify-center bg-black bg-opacity-85>
+      <h1 text-7xl>
+        Error {{ error.statusCode }}
+      </h1>
+      <p mt-4 text-2xl text-gray-400>
+        {{ error.statusMessage }}
+      </p>
+      <Button variant="grayDefault" size="lg" text-size="base" mt-6>
+        <NuxtLink to="/">
+          Go back home
+        </NuxtLink>
+      </Button>
+    </div>
+  </div>
+</template>
+
+<script setup lang="ts">
+import type { NuxtError } from "#app";
+
+// eslint-disable-next-line unused-imports/no-unused-vars
+const props = defineProps({
+  error: Object as () => NuxtError
+});
+
+useHead({
```

Comment:
To reduce boilerplate try moving it into default layout. But i'm not sure if it will work...

Reply:
what do you mean?

Reply:
Now we have duplicated `useHead`, I think you can solve it by creating default layout https://nuxt.com/docs/guide/directory-structure/layouts